utils/Cargo.toml

```diff
@@ -46,24 +46,24 @@ nimiq-keys = { path = "../keys" }
 [features]
 crc = []
 otp = ["beserial", "clear_on_drop", "nimiq-hash", "rand"]
-key-store = ["thiserror", "beserial"]
+key-store = ["beserial", "log", "thiserror"]
 iterators = []
 locking = ["futures", "parking_lot"]
-merkle = ["beserial", "beserial_derive", "nimiq-hash", "math", "nimiq-collections/bitset"]
+merkle = ["beserial", "beserial_derive", "math", "nimiq-collections/bitset", "nimiq-hash"]
 mutable-once = []
-observer = ["tokio-02"]
+observer = ["log", "tokio-02"]
```

Comment:
Most of the changes in this file is to alphabetically order the dependencies but note that here I'm actually adding `log` to the `observer` feature.